gnoblog-cli/cmd/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"github.com/peterbourgon/ff/v3/ffcli"
+	"os"
+)
+
+func main() {
+	// Create the root command
+	cmd := &ffcli.Command{
+		ShortUsage: "<subcommand> [flags] [<arg>...]",
+		LongHelp:   "The CLI for easy use of the r/blog realm",
+		FlagSet:    nil,
+		Exec: func(_ context.Context, _ []string) error {
+			return flag.ErrHelp
+		},
+	}
+
+	// Add the subcommands
+	cmd.Subcommands = []*ffcli.Command{
+		newPostCommand(),
+	}
+
+	// Run root command
+	if err := cmd.ParseAndRun(context.Background(), os.Args[1:]); err != nil {
+		_, _ = fmt.Fprintf(os.Stderr, "%+v", err)
```

Comment:
Why does this need to be set to ignored values?

Reply:
It doesn't, but the values are not used anyways, since if the command fails it will just exit.

Reply:
```suggestion
		fmt.Fprintf(os.Stderr, "%+v", err)
```
So then no need to set it to a value at all, right?

Reply:
Unfortunately the linter doesn't like it :)